cvat-ui/src/components/analytics-report/analytics-page-header.tsx

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Title from 'antd/lib/typography/Title';
+import Button from 'antd/lib/button';
+import DatePicker from 'antd/lib/date-picker';
+import { Row, Col } from 'antd/lib/grid';
+import { DownloadOutlined } from '@ant-design/icons';
+
+import { Project, Task, Job } from 'cvat-core-wrapper';
+
+interface Props {
+    onExportEvents(): void;
+    onUpdateTimePeriod(from: Date | null, to: Date | null): void;
+    resource: Project | Task | Job | null;
```

Comment:
Why it can be null? It feels we dont need to render this content if there is no resource